build/ci/get_image_tags_from_branch.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -xe
+GIT_BRANCH=$1
```

Comment:
I think this parameter should be last, to be consistent with the output.
I think the reason this is currently first, is because of the current filename.
I think the filename should be something like `compose_image_tag.sh` (not `get`, not plural `tags`, not only `from_branch`)

Reply:
it's first because of this line
https://github.com/IBM/ibm-block-csi-operator/blob/aba7bb060c67f23e38e5089145730a6a5b3e84ee/build/ci/github_actions/driver/get_driver_images_tag_from_branch.sh#L6

Reply:
looking at one line after the one you linked - how does it get a `target_specific_tag` without the other arguments?

Reply:
I still think `compose_image_tags.sh` would be a better name

Reply:
> so why is it [called `target_specific_tag`](https://github.com/IBM/ibm-block-csi-operator/blob/aba7bb060c67f23e38e5089145730a6a5b3e84ee/build/ci/github_actions/driver/get_driver_images_tag_from_branch.sh#L7)?
`triggering_branch_tag` is better?

Reply:
no I just wasn't sure of the name

Reply:
could `branch_tag` be `branch_image_tag`?

Reply:
and the first [branch_image_tag](https://github.com/IBM/ibm-block-csi-operator/blob/3ee4e462f44ddf3dd64e2aeac7eb87646c56131a/build/ci/github_actions/driver/get_driver_images_tag_from_branch.sh#L6) be `branch_image_tags` (plural)?

Reply:
done